improver/cli/__init__.py

```diff
@@ -459,15 +504,19 @@ def execute_command(dispatcher, prog_name, *args, verbose=False, dry_run=False):
             arg = ObjectAsStr(arg)
         args[i] = arg
 
-    if verbose or dry_run:
-        print(" ".join([shlex.quote(x) for x in (prog_name, *args)]))
+    msg = " ".join([shlex.quote(x) for x in (prog_name, *args)])
     if dry_run:
+        if verbose:
+            print(msg)
```

Comment:
We need a logging approach in improver too.. I wonder if we should set up something basic, then it might make a nice first task going through a few CLIs and adding some logging calls?